Use $(MKOCTFILE) variable instead of hardcoded "mkoctfile"

$ diff -Naup ./src/Makefile.orig ./src/Makefile
--- ./src/Makefile.orig	2021-07-20 07:01:49.000000000 +0200
+++ ./src/Makefile	2021-07-26 09:59:22.601812600 +0200
@@ -200,10 +200,10 @@ endif
 
 ifneq (:,/usr/bin/octave)
 %.mex: %.cpp
-	mkoctfile $(DEFINES) -I.. -v -g --mex "$<" -L.. -lbiosig -o "$@"
+	$(MKOCTFILE) $(DEFINES) -I.. -v -g --mex "$<" -L.. -lbiosig -o "$@"
 
 %.oct: %.cpp
-	mkoctfile $(DEFINES) -I.. "$<" -L.. -lbiosig -o "$@"
+	$(MKOCTFILE) $(DEFINES) -I.. "$<" -L.. -lbiosig -o "$@"
 endif
 
 %.mexw32: %.cpp
@@ -219,4 +219,3 @@ endif
 
 clean:
 	-$(DELETE) *.o *.obj *.o64 core octave-core *.oct *.mex* mexSOPEN.cpp
-
